platform/vs/docker-sonic-vs/start.sh

```diff
@@ -29,7 +29,8 @@ rm -f /var/run/rsyslogd.pid
 
 supervisorctl start rsyslogd
 
-mkdir -p /var/run/redis
+mkdir -p /var/run/redis/sonic-db
+cp /etc/default/sonic-db/database_config.json /var/run/redis/sonic-db/
```

Comment:
>database_config.json [](start = 25, length = 20)

redis-server is not reading this file right now.

Reply:
yes. Let's replace the core C++ APIs and redis-cli scripts first. Then I'll go back to change the vs images redis-server stuff. For now , this change only make sure the vs tests are not broken when doing the replacement.

Reply:
This is still applicable. If I unblock now, it will be forgotten easily.

---
In reply to: [346558103](https://github.com/Azure/sonic-buildimage/pull/3757#discussion_r346558103) [](ancestors = 346558103)

Reply:
we can open an issue and assign it to me for reminder, at this moment I don't want to do this part first. Later when we change the vs tests, this is a must do stuff, it cannot be forgotten.

Reply:
I'd like to finish the core stuff first(those APIs in swss and redis-cli replacement), then the image can be run on DUT and we can start testing on local . After that we can focus on vs tests.